packages/api-gateway/src/api/catalogApiConverter.ts

```diff
@@ -5,12 +5,15 @@ import {
 } from "pagopa-interop-api-clients";
 import { Logger } from "pagopa-interop-commons";
 import {
-  assertNonDraftDescriptor,
+  assertNotValidDescriptor,
   assertRegistryAttributeExists,
 } from "../services/validators.js";
 
-export type NonDraftCatalogApiDescriptor = catalogApi.EServiceDescriptor & {
-  state: Exclude<catalogApi.EServiceDescriptorState, "DRAFT">;
+export type validCatalogApiDescriptor = catalogApi.EServiceDescriptor & {
```

Comment:
Capitalize the type definition

```suggestion
export type ValidCatalogApiDescriptor = catalogApi.EServiceDescriptor & {
```